### PR TITLE
Riley McCullough 2207 BEE

### PR DESCRIPTION
**Pull request by Rileybmcc**
I completed User Story 1 and addressed edge cases.

Due to the time constraint I did not have time to write tests but thoroughly tested it in local host

### CODE REVIEW

File: app/services/quotable_service.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class QuotableService
```
Comment (epintozzi) [inline app/services/quotable_service.rb:1]: Service looks good 👍🏻

File: app/facades/quotable_facade.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class QuotableFacade
+
+  def self.search_for_quotes(query)
+    # require "pry"; binding.pry
+    json = QuotableService.search_quotes(query)
+    if json[:results].count > 0
+      json[:results][0..9].map {|quote| Quote.new(quote, json[:count])}
+    else
+      [Quote.new({results: {content: [], tags: [], author: []}}, json[:count])]
+    end
```
Comment (epintozzi) [inline app/facades/quotable_facade.rb:10]: This is an interesting implementation. Were you finding that you were getting errors when no results were returned?

File: app/facades/quotable_facade.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class QuotableFacade
+
+  def self.search_for_quotes(query)
+    # require "pry"; binding.pry
+    json = QuotableService.search_quotes(query)
+    if json[:results].count > 0
+      json[:results][0..9].map {|quote| Quote.new(quote, json[:count])}
```
Comment (epintozzi) [inline app/facades/quotable_facade.rb:7]: Nice solution to include the count in your poro. It's much more efficient to collect all the data you need at one time rather than making multiple API calls. Good work!

**Review by epintozzi — COMMENTED** [review]
Overall this looks good! I know you mentioned time constraints for lack of testing so definitely keep practicing how you would test these classes so you are more comfortable with it for your projects and final assessment.
